training/tf/tfprocess.py

```diff
@@ -301,3 +308,36 @@ def construct_net(self, planes):
         h_fc3 = tf.nn.tanh(tf.add(tf.matmul(h_fc2, W_fc3), b_fc3))
 
         return h_fc1, h_fc3
+
+
+class StatisticsCollector(object):
```

Comment:
This code appears to be lifted out of another program without attribution or conserving the license. If this is from @brilee Mugo, we can use it because Mugo's license is compatible with ours (Apache 2.0 and GPL3). But it should be moved to a separate source file so you properly attribute the copyright and mention the license. Probably also need to update leela-zero's README.md which lists the licenses.